sdk/loadtestservice/Azure.Developer.LoadTesting/Hero/Program.cs

```diff
@@ -0,0 +1,200 @@
+            catch (Exception e)
+            {
+                Console.WriteLine(String.Format("Error : ", e.Message));
+            }
+
+
+            // wait for file to complete Validation
+            try
+            {
+                TestFileValidationStatus testFileValidationStatus = loadTestAdministrationClient.BeginGetTestScriptValidationStatus(testId);
+
+                Console.WriteLine(testFileValidationStatus.ToString());
+
+                if(testFileValidationStatus != TestFileValidationStatus.ValidationSuccess)
+                {
+                    // if file validaion failed, then stop the process
+                    return;
+                }
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine(String.Format("Error : ", e.Message));
+            }
+
+
+            // connecting app component
+            try
+            {
+                var data = new
+                {
+                    testid = testId,
+                    name = "New App Component",
+                    value = new
+                    {
+                        appComponentConnectionString = new
+                        {
+                            resourceId = appComponentConnectionString,
+                            resourceName = "App-Service-Sample-Demo",
+                            resourceType = "Microsoft.Web/sites",
+                            subscriptionId = subscriptionId
+                        }
+                    }
+                };
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine(String.Format("Error : ", e.Message));
+            }
+
+
+            // running test
+            try
+            {
+                var data = new
+                {
+                    testid = testId,
+                    displayName = "Some display name"
+                };
+
+                Response response = loadTestRunClient.CreateOrUpdateTestRun(testRunId, RequestContent.Create(data));
+            }
+            catch(Exception e)
+            {
+                Console.WriteLine(String.Format("Error : ", e.Message));
+            }
+
+
+            // waiting for test to complete
+            try
+            {
+                TestRunStatus testRunStatus = loadTestRunClient.BeginTestRunStatus(testRunId);
+                Console.WriteLine(testRunStatus.ToString());
+
+                if(testRunStatus!= TestRunStatus.ValidationSuccess)
+                {
+                    return;
+                }
+            }
+            catch(Exception e)
+            {
+                Console.WriteLine(String.Format("Error : ", e.Message));
+            }
+
+            // getting matrics from loadtest
+            try
+            {
+                Response responseGetTestRun = loadTestRunClient.GetTestRun(testRunId);
+                JsonDocument responseGetTestRunJson = JsonDocument.Parse(responseGetTestRun.Content.ToString());
```

Comment:
@prativen I am not very clear about this comment, can you please help me here with more context?

Reply:
@NiveditJain This is regarding the variable name responseGetTestRunJson; Since we are assigning it the JSONDocument version of testRunResponse.Content; I was checking what is the content? If the content property is the testRun object itself, suggest renaming it to testRunJson else testRunContentJson